### PR TITLE
Update -data-release.yml

### LINKED ISSUE #1: [NSSDCA] Enter_Bundle_LIDVID_Here
**Delivering Node**: _please enter your DN here for record keeping purposes_

**NSSDCA Delivery Package**: _please upload the files output by PDS Deep Archive as a TAR.GZ or ZIP file here, or supply a URL to download from_

### LINKED ISSUE #2: [NSSDCA] urn:nasa:pds:insight-ifg-cruise::1.0
from PPI node.  If a delivery ID is needed, how about PPI_test-001_2020-07-14

[PPI_insight-ifg-cruise_test-1.zip](https://github.com/NASA-PDS/pdsen-operations/files/4922177/PPI_insight-ifg-cruise_test-1.zip)

### PR DESCRIPTION
**Pull request by rchenatjpl**
Change Access URL to a textarea to allow multiple lines, which will help collections that use their own URLs and not the bundle's.

### 🤖 AI Assistance Disclosure
- [x] No AI assistance used
- [ ] AI used for light assistance (e.g., suggestions, refactoring, documentation help, minor edits)
- [ ] AI used for moderate content generation (AI generated some code or logic, but the developer authored or heavily revised the majority)
- [ ] AI generated substantial portions of this code

Estimated % of code influenced by AI: ___ %

## 🤓 Reviewer Checklist

*Reviewers: Please verify the following before approving this pull request.*

### Documentation and PR Content
- [ ] **Documentation:** README, Wiki, or inline documentation (Sphinx, Javadoc, Docstrings) have been updated to reflect these changes.
- [ ] **Issue Traceability:** The PR is linked to a valid GitHub Issue
- [ ] **PR Title:** The PR title is "user-friendly" clearly identifying what is being fixed or the new feature being added, that if you saw it in the Release Notes for a tool, you would be able to get the gist of what was done.

### Security & Quality
- [ ] **SonarCloud:** Confirmed no new High or Critical security findings.
- [ ] **Secrets Detection:** Verified that the Secrets Detection scan passed and no sensitive information (keys, tokens, PII) is exposed.
- [ ] **Code Quality:** Code follows organization style guidelines and best practices for the specific language (e.g., PEP 8, Google Java Style).

### Testing & Validation
- [ ] **Test Accuracy:** Verified that test data is accurate, representative of real-world PDS4 scenarios, and sufficient for the logic being tested.
- [ ] **Coverage:** Automated tests cover new logic and edge cases.
- [ ] **Local Verification:** (If applicable) Successfully built and ran the changes in a local or staging environment.

### Maintenance
- [ ] **Backward Compatibility:** Confirmed that these changes do not break existing downstream dependencies or API contracts (or that breaking changes are clearly documented).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/-data-release.yml [hunk 1]
```diff
@@ -46,14 +46,14 @@ body:
...
     validations:
       required: true
 
-  - type: input
+  - type: textarea
     attributes:
-      label: "Access URL"
-      description: "URL to give users searching for this data"
+      label: "Access URL for Search Results"
+      description: "Line 1: URL of the bundle (or dataset). Lines 2+ (optional): If a collection resolves to a URL(s) other than Line 1's, each Line 2+ is a pair: LID, URL for that LID."
```
[the remark this answers is not in the document]
Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: The "(LID, comma, space, URL)" seems extraneous since there's an e.g. (example). Overall, I like the clarification. Richard, does it seem too rigid to you?

File: .github/ISSUE_TEMPLATE/-data-release.yml [hunk 2]
```diff
@@ -46,14 +46,14 @@ body:
   - type: textarea
     attributes:
       label: "Data To Be Ingested"
-      description: "Data to be ingested into the Legacy PDS Registry for release. For PDS4: URLs to the bundles, collections, and all associated inventories; or attach a zip of all applicable files. For PDS3: URLs to the catalog files that have changed or attach a zip of all applicable files."
+      description: "One URL to the standard-ish top-level PDS4 directory with bundle and collections (or PDS3 dir with voldesc and catalog files). Or a zip of such files. Or multiple URLs"
```
Comment (c-suh) [inline .github/ISSUE_TEMPLATE/-data-release.yml]: Any clarification on if there's multiple URLs? Because otherwise, isn't it back to what nodes are just doing now already?

File: .github/ISSUE_TEMPLATE/-data-release.yml [hunk 3]
```diff
@@ -46,14 +46,14 @@ body:
...
     validations:
       required: true
 
-  - type: input
+  - type: textarea
     attributes:
-      label: "Access URL"
-      description: "URL to give users searching for this data"
+      label: "Access URL for Search Results"
```
Comment (c-suh) [inline .github/ISSUE_TEMPLATE/-data-release.yml]: 👍

**Review by c-suh — COMMENTED** [review]
1 question (line 49), 1 thumbs up (line 55), and 1 comment (line 56) to Copilot's suggestion.

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jordanpadams) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: @rchenatjpl @c-suh can we iterate over these comments and merge this sooner rather than later? looks like the PR has been open for 14 days.

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 2 above)
Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: Perhaps, multiple URLS _to the top-level directory_? If so, I'd say they should open another ticket, so I propose to just remove this last part.

Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: Proposed rewrite (for brevity): "For the data to ingest, ideally, one crawlable URL. Otherwise, a zip file."

Or, for completeness, append: "...  For PDS4: the top-level directory of the bundle or collection. For PDS3: the directory of the catalog and voldesc files."

Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: @rchenatjpl ☝️

**Review by c-suh — COMMENTED** [review]
See specific comments.

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: @tloubrieu-jpl's suggestion: leave this description as is, and add another field, which is not required, along the lines of "Additional Access URLs for Search Results".

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 3 above)
Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: Jordan says 👎  and proposes "...for Landing Page"

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (c-suh) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: Jordan's suggestion: specify "primary" in the original description.

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 2 above)
Comment (jordanpadams) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: @copilot can you make these updates per these comments?

File: .github/ISSUE_TEMPLATE/-data-release.yml (hunk 3 above)
Comment (jordanpadams) [reply .github/ISSUE_TEMPLATE/-data-release.yml]: updates done

**Review by c-suh — APPROVED** [review]
Approving after changes per previous review comments.

**Comment by rchenatjpl** [comment]
@c-suh  For "Data To Be Ingested", please allow only 1 line of input rather than a text area. Sorry to request that here.

**Comment by rchenatjpl** [comment]
And for "Additional Access URL", I'd say change "(optional) If a collection resolves to a URL(s) other than Primary Access URL, add comma-separated URLs like LID,URL" to "(optional) If any collections or the bundle resolves to additional URLs, add list of comma-separated LID,URL"
